### PR TITLE
Split trade pool view

### PR DESCRIPTION
**Pull request by yamijuan**
image

image

This is on top of #19 so it needs to be merged first

### CODE REVIEW

**Comment by SensyCheung** [comment]
This UI(Split trade pool view) isn't good.

File: src/context/market.tsx [hunk 1]
```diff
@@ -404,7 +405,7 @@ function createEnrichedPools(
       let volume = 0;
       let volume24h =
         baseMid * (poolData?.get(p.pubkeys.mint.toBase58())?.volume24hA || 0);
-      let fees24h = volume24h * 0.0025;
+      let fees24h = volume24h * (LIQUIDITY_PROVIDER_FEE - SERUM_FEE);
```
Comment (bartosz-lipinski) [inline src/context/market.tsx:408]: 👍  awesome

File: src/components/trade/index.tsx [hunk 2]
```diff
@@ -169,148 +180,163 @@ export const TradeEntry = () => {
...
-    if(priceAccount === B.mintAddress) {
-      setExchangeRate(parseFloat(B.amount) / parseFloat(A.amount))
+    if (priceAccount === B.mintAddress) {
+      setExchangeRate(parseFloat(B.amount) / parseFloat(A.amount));
     } else {
-      setExchangeRate(parseFloat(A.amount) / parseFloat(B.amount))
+      setExchangeRate(parseFloat(A.amount) / parseFloat(B.amount));
     }
-  }, [A, B, priceAccount])
+  }, [B, slippage, pool, enriched, priceAccount]);
```
Comment (bartosz-lipinski) [inline src/components/trade/index.tsx]: why is A not in the deps list?

Comment (yamijuan) [reply src/components/trade/index.tsx]: There was a bug in the text. All the numbers flashed with weird values. So the problem is when you change A, B is changed Async. So if you had any value on B, the rate A / B wasn't updated right away and the numbers changed till B is updated.

Comment (yamijuan) [reply src/components/trade/index.tsx]: I'm gonna Add `A` again to the deps, and work on the issue

**Comment by bartosz-lipinski** [comment]
> ol view) isn't good.

what would you like to change?

**Pull request merged**